### PR TITLE
Homework8: Yanislav Terentev

### PR DESCRIPTION
**Pull request by YanislavTerentev**

### CODE REVIEW

File: Homework_8/consistency.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    if compare_elements(array):
+        result = True
+    else:
+        for i in range(len(array)):
+            copy_array = array.copy()
+            del copy_array[i]
+            if compare_elements(copy_array):
+                result = True
+                break
+    print(result)
```
Comment (Stml89) [inline Homework_8/consistency.py]: return result, not print it

File: Homework_8/consistency.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+solution([1, 2, 3])
```
Comment (Stml89) [inline Homework_8/consistency.py]: use `assert` instead of `print()`

File: Homework_8/number_opposite.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+"""This function print opposite number"""
+
+
+def number_opposite(n, first_number):
+    """This function print opposite number"""
+    if first_number > n/2:
```
Comment (Stml89) [inline Homework_8/number_opposite.py:6]: Looks better
```
    if first_number > n/2:
        return ...
    return ...
```

**Review by Stml89 — CHANGES_REQUESTED** [review]

File: Homework_8/consistency.py (hunk 1 above)
Comment (YanislavTerentev) [reply Homework_8/consistency.py]: Done

File: Homework_8/consistency.py (hunk 2 above)
Comment (YanislavTerentev) [reply Homework_8/consistency.py]: Done

File: Homework_8/number_opposite.py (hunk 3 above)
Comment (YanislavTerentev) [reply Homework_8/number_opposite.py:6]: Done

**Review by Stml89 — APPROVED** [review]

**Pull request merged**